examples/synthesis_json.py

```diff
@@ -113,31 +114,33 @@ def transform_package(mtype, files, components, feature_list):
     return data_dict
 
 
-def get_mtype(filename, sep='_'):
+def get_mtype_from_filename(filename, sep='_'):
     '''Get mtype of a morphology file from file name
 
     Assumes file name has structure 'a/b/c/d/mtype_xyx.abc'
     '''
     return os.path.basename(filename).split(sep)[0]
 
+
 if __name__ == '__main__':
     args = parse_args()
 
-    d_path = args.datapath
+    data_dirs = args.datapaths
 
     mtype_files = defaultdict(list)
 
-    for f in get_morph_files(d_path):
-        mtype_files[get_mtype(f)].append(f)
-
     flist = [["soma_radius", "radius", "soma", None, None, None],
              ["n_neurites", "number", "basal_dendrite", 1, None,
```

Comment:
For my education: what do "n_neurites" and "number" mean here?

(n_neurites is probably number of neurites, but what is number?)